courses/free-courses-kn.md

```diff
@@ -0,0 +1,36 @@
+### Index
+
+* [C](#c)
+* [HTML and CSS](#html-and-css)
+* [JavaScript](#javascript)
+* [PLC](#plc)
+* [Python](#python)
+
+
+### C
+
+* [Complete C Language Course in KANNADA](https://youtu.be/OvyNtFgn8T0?si=Bb0rLk3Cfl0zMcaU) - The Coding Class: Yuvaraj Madha
+* [C programming in Kannada](https://youtu.be/MhRzCX18mDE?si=vp89xBkxJxB1PdfZ) - script kiddie
+
+
+### HTML and CSS
+
+* [CSS in Kannada (ಕನ್ನಡದಲ್ಲಿ) - Beginner to Advanced.](https://youtu.be/wAqQScpoOMQ?si=Lx8zLwW1lm-sqBeL) - DEBUG CODING
+* [HTML and CSS in Kannada || Web Designing in Kannada || CSS in Kannada || Html in Kannada](https://youtube.com/playlist?list=PLBVoFEaXQzK3F2BN7ZbXb_EpnDMAlJsPo&si=xrva9ZdT93aR4n2N) - Learn Tech in Kannada
```

Comment:
remove the tracking parameter.

Reply:
@eshellman I have made necessary changes in the Pull request.Please add me as assignee and add hacktoberfest and hacktoberfest-accepted label and please merge this pr
Thank you.